tests/backstop_data/engine_scripts/puppet/contacts/contact-dashboard.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const Utility = require('./../utility.js');
+
+module.exports = async (page, scenario, vp) => {
+  const utility = new Utility(page, scenario, vp);
+
+  await page.type('#sort_name', 'Betty Adams');
```

Comment:
Dont wait for a specifc name, instead the id of the contact should be coming from the scenario file(same as CiviHR), we should to wait for something using that ID.

Reply:
As discussed in chat we are going to keep this for now, and move to a `cli` based approach as we do in CiviHR. Since this has been done for C51-393 we will review those changes there.